src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/coreperson/CorePersonMapping.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.coreperson
+
+import org.springframework.data.annotation.Id
+import java.time.LocalDateTime
+
+class CorePersonMapping(
+  @Id
+  val cprId: String,
+  val offenderNo: String,
```

Comment:
rather than `prisonNumber`?  What is used in the rest of this project

Reply:
I was toying with swapping this to prisonNumber - offenderNo is used elsewhere, so I used that.  I will update to prisonNumber as that seems to be the way forward - and so new migrations/syncs can use that too